packages/tools/src/tools/annotation/PlanarFreehandROITool.ts

```diff
@@ -745,9 +747,25 @@ class PlanarFreehandROITool extends AnnotationTool {
 
       const { imageData, metadata } = image;
       const canvasCoordinates = points.map((p) => viewport.worldToCanvas(p));
+
+      const canvasPoint = canvasCoordinates[0];
+      const deltaX = viewport.canvasToWorld([
+        canvasPoint[0] + 1,
```

Comment:
why there is a +1?

Reply:
The aim is to convert a step in the x direction to mm. We can pick any arbitrary point as our origin, take one step in the x and one step in the y and use those to calculate the actual mm spacing in the x and y canvas axes.

Reply:
I see, then the code benefits from this comment above, can you add it on top of the code?